### ISSUE TITLE
test/lint.sh should check for 2 space indent

### ISSUE THREAD

**Issue by andychu** [oilshell/oil#408]
Except for the `opy/` dir, etc.

**Comment by drwilly** [comment]
oops

**Comment by andychu** [comment]
Yeah I prefer to keep it consistent :)  But if you think there is more to do on find/xargs, then I wouldn't worry about it right now.

At some point I will try to integrate it with `bin/oil.py`, MyPy, lint, etc.  And it actually came up for xargs on #167 and #411 -- it would be nice to use GNU-compatible xargs on other platforms to run spec tests.

But for now it's OK if you use your own style, since you're the only one working in there.  When it's integrated with the rest of the codebase that should change.

Do you have any future plans for find/xargs ?  I can start testing more and looking for bugs.

**Comment by drwilly** [comment]
Well, converting from Tabs to Space is only one `sed` away.

> Do you have any future plans for find/xargs ? I can start testing more and looking for bugs.

Not really, no. Also from august onwards I will have a lot less free time on my hands, so there probably wont be major changes from my side.
I was hoping find/xargs were in a state where the internal "framework" was in an acceptable state and development could continue incrementally, one patch at a time.

**Comment by andychu** [comment]
Well I also disabled the rest of `test/lint.sh` because there were MyPy errors.  I don't think your type annotations were ever checked.

I guess I will try a few things.  Let me file a new bug for it.

**Comment by drwilly** [comment]
No, I didn't have MyPy installed, so those are all basically comments.
